mount/osf1-lib.pl

```diff
@@ -262,9 +262,9 @@ sub disk_space
 if ($_[0] eq "fd" || $_[0] eq "proc" || $_[0] eq "swap" || $_[0] eq "autofs") {
 	return ();
 	}
-if (&backquote_command("df -k ".quotemeta($_[1]), 1) =~
-    /Mounted on\n\S+\s+(\S+)\s+\S+\s+(\S+)/) {
-	return ($1, $2);
+my $out = &backquote_command("df -k ".quotemeta($_[1]), 1);
```

Comment:
Are you sure this OS has the same `df` output as Linux?

Reply:
Just checked. The output is not the same, as it also includes `inodes`.  However, this regex will work as well.